### PR TITLE
Try fix for damageprofiler sometimes needing a fai file

### PR DESCRIPTION
**Pull request by jfy133**
Add fix for damageprofiler edge case requiring FAI file.

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](<https://github.com/>nf-core/eager/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/eager _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - Changed endors.py license from GPL to MIT (by @aidaanva)
 - Removed erroneous R2 in single-end example in input TSV of usage docs (by @aidaanva)
+- Fixed edge case of DamageProfiler occasionally requiring FASTA index (♥ to Asmaa Ali for reporting)
```
Comment (jfy133) [inline CHANGELOG.md:14]:
```suggestion
- Fixed edge case of DamageProfiler occasionally requiring FASTA index (♥ to @asmaa-a-abdelwahab for reporting)
```

**Comment by jfy133** [comment]
i have confirmation from @asmaa-a-abdelwahab that this works it seems (at least doesn't get this particular error)

**Review by apeltzer — APPROVED** [review]
